tutorials/nemo-retriever-synthetic-data-generation/main.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import importlib
+import os
+import shutil
+from typing import Any, List
+
+from tqdm.dask import TqdmCallback
+
+from nemo_curator import AsyncOpenAIClient, ScoreFilter, Sequential
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.filters import AnswerabilityFilter, EasinessFilter
+from nemo_curator.modules.filter import Score, ScoreFilter
+
+config = importlib.import_module(
```

Comment:
Does the following import not work?
```python
from config import config
```

`tutorials/peft-curation` does something similar [here](https://github.com/NVIDIA/NeMo-Curator/blob/main/tutorials/peft-curation/main.py#L21).

Reply:
fixed